src/test/java/teammates/test/pageobjects/InstructorCommentsPage.java

```diff
@@ -209,6 +209,15 @@ public void clickResponseCommentDelete(int sessionIdx, int questionIdx, int resp
     }
     
     /**
+     * Clicks 'Comments for students' panel heading of the comment panel to either expand/collapse the panel body.
+     */
+    public void clickSingleCommentsPanelHeading() {
```

Comment:
I think it would be better to name this method as `clickStudentCommentsPanelHeading` as `clickSingleCommentsPaneHeading` could refer to any panel